app/src/server/trpc/routers/service.ts

```diff
@@ -1,9 +1,122 @@
 import { publicProcedure, router } from '../trpc';
-import { createDraft } from '@/server/db/dbOperations';
+import { createDraft, getDraftById, getDraftByUrl } from '@/server/db/dbOperations';
 import { modifyResourceToDraft } from '@/util/modifyResourceFields';
-import { FhirArtifact } from '@/util/types/fhir';
+import { ArtifactResourceType, FhirArtifact } from '@/util/types/fhir';
 import { z } from 'zod';
 import { TRPCError } from '@trpc/server';
+import { DateTime } from 'luxon';
+
+export interface ReleaseArtifactInfo extends ArtifactInfo {
+  res: Response;
+  location: string | null;
+}
+
+export type ArtifactInfo = {
+  resourceType: 'Measure' | 'Library';
+  id: string;
+};
+
+async function draftChildren(relatedArtifacts: fhir4.RelatedArtifact[]) {
```

Comment:
I like this idea! What would the most helpful logs look like? We log whenever a draft is created or released (including child artifacts), what else would you like to see included?

Reply:
For now I think it would be most helpful to put logs in the `draftChildren` and `releaseChildren` functions specifically logging that a child artifact is being drafted/released, perhaps along with their id. I don't think we need to add super extensive logging since we do have logging for the service operations already - would be helpful to have a few additional logs indicating whether a resource is a child artifact or not so that there's a little more context to the logs.

Reply:
This was the one thing I had a lot of trouble with adding and figured I could either continue working on it after I restructured stuff or even put it in a different task...I tried loggerLink and it looks like it just formatted it in the browser console...but we want logs in the console with the server logs, right?